### ISSUE TITLE
Heart canisters vanished after update

### ISSUE THREAD

**Issue by TheRealDadbeard** [Radient-Sora/Baubley-Heart-Canisters#23]
So after updating all my canisters just vanished in my baubles menu. I still have the yellow hearts but don't know what is going on. I have it enabled so that you can wear them without the amulet.

Was that supposed to happen?

**Comment by Radient-Sora** [comment]
Yeah so it's 2.0 Beta cause it's basically a re-write that needed to happen . I'm sorry you lost your stuff inform others if need be.

**Comment by TheRealDadbeard** [comment]
Yeah just saw the new config location file, thats why I was so confused. It's okay, I'll just give me and the other players new ones :)

**Comment by Radient-Sora** [comment]
Awesome 👍 I am gonna leave this open just so others don't report it. But this issue is (closed)
